worker/buildbot_worker/base.py

```diff
@@ -315,7 +315,10 @@ def remote_getWorkerInfo(self):
                 if os.path.isfile(filename):
                     files[f] = open(filename, "r").read()
         if not self.numcpus:
-            self.numcpus = multiprocessing.cpu_count()
+            try:
+                self.numcpus = multiprocessing.cpu_count()
+            except NotImplementedError:
+                self.numcpus = 1
```

Comment:
I think we should at least give a warning in the log that numcpus is incorrect for this worker.